SECURITY.md

```diff
@@ -1,58 +1,57 @@
 # Security Policy
 ## Supported Versions
-Current supported versions for Gravitee Access Management:
-| Version | Supported          |
-| ------- | ------------------ |
-| LTS:    |                    |
-| 3.10.x  | :white_check_mark: |
-| 3.5.x   | :white_check_mark: |
-|         |                    |
-| < 3.10  | :x:                |
-| < 3.5   | :x:                |
+Current supported versions for Gravitee APIm are available in our Documentation page available at:
+https://docs.gravitee.io/apim/3.x/apim_changelog.html
```

Comment:
https://docs.gravitee.io/am/current/am_changelog.html

Reply:
Not sure this page shows the supported versions